common/v2/utils/convert.ts

```diff
@@ -20,3 +20,23 @@ export const convertToFiat = (userViewableBalance: number, rate: number = 1): nu
 
 export const weiToFloat = (wei: BigNumber, decimal?: number): number =>
   parseFloat(fromTokenBase(new BN(wei.toString()), decimal || DEFAULT_ASSET_DECIMAL));
+
+export const withCommission = ({
+  amount,
+  rate,
+  substract
+}: {
+  amount: number;
```

Comment:
SHOULD be a `ethers.js` BigNumber

Reply:
it's going to change when I will be using convertToFiat :)